tests/voiceflowToBotFormat.test.js

```diff
@@ -0,0 +1,75 @@
+const convert = require("../voiceflowToBotFormat")
+const largeDiagrams = require("./testDiagrams")
+
+test("Empty String as Input", () => {
+    convert("").then(resp => {
+        expect(resp).toBeNull()
+    })
+})
+
+
+test("Undefined Diagram as Input", () => {
+    convert(undefined).then(resp => {
+        expect(resp).toBeNull()
+    })
+})
+
+test("Empty Diagram type as Input", () => {
+    diagram = {}
+    convert(diagram).then(resp => {
+        expect(resp).toBeNull()
+    })
+})
+
+test("Partial Diagram as Input", () => {
+    diagram = {"project": {"name": "test"}}
+    convert(diagram).then(resp => {
+        expect(resp).toBeNull()
+    })
+})
+
+test("Valid Diagram as Input", () => {
+    diagram = JSON.parse(largeDiagrams.getValidDiagram())
+    convert(diagram).then(resp => {
+        expect(resp).toStrictEqual(JSON.parse(largeDiagrams.getValidDiagramUBF()))
+    })
+})
+
+test("Valid Diagram with additional nodes as Input", () => {
+    diagram = JSON.parse(largeDiagrams.getValidDiagramExtraNodes())
+    convert(diagram).then(resp => {
+        expect(resp).toStrictEqual(JSON.parse(largeDiagrams.getValidDiagramExtraNodesUBF()))
+    })
+})
+
+test("Invalid Diagram as Input", () => {
+    diagram = {"project": {"random key": "test"}}
```

Comment:
I would also try passing a valid diagram and then adding a random key (so testing if it rejects new keys)